package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stellar-sdk",
-  "version": "9.0.0",
+  "version": "8.2.5",
```

Comment:
The current version is already 8.2.5:
https://github.com/stellar/js-stellar-sdk/blob/8e85393150964057d7768755ec62b2a273c98fbf/package.json#L3

Shouldn't we use `8.2.6` here?

Reply:
I don't think npm lets you override a version so we should always increment versions.

Reply:
This is intentional; I'm restoring the version that's in master so it doesn't change when we merge #693. In other words, merging #693 should not release to npm yet.